### ISSUE TITLE
Datos desbalanceados

### ISSUE THREAD

**Issue by nicolaszanni** [IIC2433/Syllabus-2021-2#49]
Hola!  Me pasa que con la cantidad de datos que hay para la prenda q, aún achicando el set test, me da un accuracy muy bueno para ambas clases, como si no hubiera efecto de datos desbalanceados. Tengo que bajar muchísimo el set de entrenamiento de q para que el accuracy para esa clase baje y tampoco baja como lo esperaba.

Una hipótesis que tengo es que las prendas p y q que escogí son muy diferentes por lo que tal vez el modelo, a pesar de tener pocos datos de q, sepa clasificar bien, pero el ejercicio de datos desbalanceados no debería mostrarse para cualquier p !=q ?

Según otra issue que vi, deberíamos tener entonces para la prenda p, 6000 en train y 1000 en test y para la q, 500 en train y 1000 en test?
No sé muy bien cual es el problema, gracias!

**Comment by gicatalan** [comment]
Sí, depende de las prendas que eliges. Prueba con pullover ('2') y y coat ('4'), que se parecen harto, ahí el problema se ve mejor.  Esperemos a ver que dice el profe o algún ayudante porque tengo las mismas dudas.

**Comment by juanreutter** [comment]
La verdad yo lo probe pa un par de prendas y bajaba un poco. Pero puedes probar la 2 y 4, o puedes seguir con cualquier clase de prendas como la que ya habias sacado, y anda viendo si el samplong te mejora o no!
